.github/workflows/check-spirv-tools-generated.yml

```
@@ -0,0 +1,125 @@
+  check-spirv-consistency:
+    name: Check SPIRV-Tools Generated Files Consistency
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          submodules: recursive
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)